lib/pbench/server/api/resources/upload_api.py

```diff
@@ -165,26 +161,31 @@ def put(self, controller):
                     hash_md5.update(chunk)
             except Exception:
                 self.logger.exception(
-                    "Tarfile upload: There was something wrong uploading {}", filename
+                    "Unexpected error uploading {}/{}/{}",
+                    username,
+                    controller_arg,
+                    filename_arg,
                 )
-                abort(500, message=f"There was something wrong uploading {filename}")
+                abort(HTTPStatus.INTERNAL_SERVER_ERROR, message="INTERNAL ERROR")
 
             if bytes_received != content_length:
-                self.logger.debug(
-                    f"Tarfile upload: Bytes received does not match with content length header value for file {filename}"
+                msg = (
+                    "Bytes received do not match Content-Length header"
+                    f" ({content_length} != {bytes_received})"
```

Comment:
This would probably be better expressed as 
```
                    f" (expected {content_length}; received {bytes_received})"
```

Reply:
Or, you could integrate it into the message:
```
                msg = (
                    "Bytes received ({bytes_received}) do not match Content-Length header ({content_length})"
```

Reply:
Moved to #2287.